tests/test_mod.py

```diff
@@ -725,6 +735,44 @@ def test_scihub_unresponsive():
             api.download_all(['8df46c9e-a20c-43db-a19a-4240c2ed3b8b'])
 
 
+def test_trigger_lta():
+    """ trigger retrieval form long time archive """
+    api = SentinelAPI("mock_user", "mock_password")
+
+    request_url = "https://scihub.copernicus.eu/apihub/odata/v1/Products('8df46c9e-a20c-43db-a19a-4240c2ed3b8b')/$value"
+
+    with requests_mock.mock() as rqst:
+        rqst.get(
+            request_url,
+            text="Mock trigger accepted", status_code=202
+        )
+        assert api._trigger_offline_retrieval(request_url) == 202
+
+    with requests_mock.mock() as rqst:
+        rqst.get(
+            request_url,
+            text="Mock trigger not accepted", status_code=503
+        )
+        with pytest.raises(SentinelAPILTAError) as excinfo:
+            api._trigger_offline_retrieval(request_url)
+
+    with requests_mock.mock() as rqst:
+        rqst.get(
+            request_url,
+            text="Mock trigger exceeds user quota", status_code=403
+        )
+        with pytest.raises(SentinelAPILTAError) as excinfo:
+            api._trigger_offline_retrieval(request_url)
+
+    with requests_mock.mock() as rqst:
+        rqst.get(
+            request_url,
+            text="Mock trigger downloading an offline file", status_code=500
+        )
+        with pytest.raises(SentinelAPILTAError) as excinfo:
+            api._trigger_offline_retrieval(request_url)
+
+
```

Comment:
Please add a higher-level test, too, that calls `api.download` to get the `break` catch covered.

Reply:
I've added a new test for `download_all` with a product in the LTA.